src/components/Login.js

```diff
@@ -153,34 +158,33 @@ class Login extends Component {
                     onClick={this.submitLogin}
                     disabled={!this.state.username || !this.state.password}
                   >
-                    LOG IN
-                  </Button>
-                </Form>
-
-                {/* form validation */}
+                    LOG IN{' '}
+                  </Button>{' '}
+                </Form>{' '}
+                {/* form validation */}{' '}
                 {this.state.usernameerror ? (
                   <Message error content="Username cannot be empty!" />
-                ) : null}
+                ) : null}{' '}
                 {this.state.passworderror ? (
                   <Message error content="Password cannot be empty!" />
-                ) : null}
+                ) : null}{' '}
                 {error && submitted ? (
                   <Message error content={this.props.loginerror.detail} />
-                ) : null}
+                ) : null}{' '}
                 <Divider />
                 <span>
-                  Don't have an account?{' '}
-                  <Link to={register()}>Register here.</Link>
-                </span>
-              </div>
-            </Grid.Column>
+                  Dont have an account{' '}
```

Comment:
```suggestion
                  Don't have an account{' '}
```
I think it's like this. Did you change this intentionally?